c2rust-analyze/src/rewrite/expr/unlower.rs

```diff
@@ -0,0 +1,379 @@
+//! Builds the *unlowering map*, which maps each piece of the MIR to the HIR `Expr` that was
```

Comment:
I would move these module docs to `fn unlower` instead, as usually I look to function documentation first, and all of this documentation makes sense for `fn unlower`, too.

Reply:
Done (also did the same in the `distribute` and `convert` modules as well)